openstack/maia/templates/etc/_maia_scrape_config.yaml.tpl

```diff
@@ -171,23 +171,54 @@
+      - '{__name__=~"^vrops_virtualmachine_memory_active_ratio", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_memory_activewrite_kilobytes", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_memory_balloning_ratio", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_memory_consumed_kilobytes", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_memory_contention_ratio", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_memory_usage_average", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_network_data_received_kilobytes_per_second", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_network_data_transmitted_kilobytes_per_second", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_network_packets_dropped.+", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_network_packets.+", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_network_usage_average_kilobytes_per_second", project!~"internal", vccluster!~".*management.*"}'
       - '{__name__=~"^vrops_virtualmachine_number_vcpus_total", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_oversized", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_oversized_vcpus", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_oversized_memory", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_runtime_connectionstate", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_runtime_powerstate", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_summary_ethernetcards", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_swapin_memory_kilobytes", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_swapped_memory_kilobytes", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_undersized", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_undersized_vcpus", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_undersized_memory", project!~"internal", vccluster!~".*management.*"}'
+      - '{__name__=~"^vrops_virtualmachine_virtual_disk_average.+", project!~"internal", vccluster!~".*management.*"}'
```

Comment:
These metrics are not available with vrops:
```
vrops_virtualmachine_virtual_disk_average_read_miliseconds
vrops_virtualmachine_virtual_disk_average_write_miliseconds
```

Reply:
That’s fair, otherwise I have no additional feedback